README.md

```diff
@@ -16,6 +16,17 @@
    ```
    - *optional*: If a package build depends on other custom build packages you can provide the `--build-dependencies` flag with a directory containing the `.deb` files of build-time dependencies
 
+The build artifacts (`deb` files and others) are placed in a `.build` directory in your package, for example in `./package-iproute2/.build/`.
+
+### Build options
+
+The `build` script takes arguments which may be used to customize the build.
```

Comment:
I would say to keep it more simple, we could remove this sentence. Context is clear that this section shows the build arguments. 

However, also works for me if it stays like this.